packages/docs/src/styles/components/footer.scss

```diff
@@ -1,2 +1,86 @@
 .c-footer {
+  .ds-footer__container {
+    margin-inline: auto;
+    max-width: $site-max-width;
+    width: 100%;
+  }
+}
+
+.c-footer__content--top,
+.c-footer__content--bottom {
+  padding: $grid-gutter-width $grid-gutter-width / 2;
+  font-size: $small-font-size;
+
+  .c-footer__content-title {
+    font-size: $h4-font-size;
+    font-weight: $font-bold;
+    margin: 0 0 $spacer-1 0;
+  }
+}
+
+.c-footer__content--top {
+  background-color: $color-gray-lightest;
+  color: $color-base;
+
+  .c-footer__feedback-links {
+    display: grid;
+    gap: $spacer-2;
+
+    @media (min-width: $width-md) {
+      grid-template-columns: repeat(3, minmax(max-content, 33.3%));
+    }
+  }
+}
+
+.c-footer__content--bottom {
+  background-color: $color-primary-darkest;
+  color: $color-white;
```

Comment:
[question] how do these styles differ from the styles that `ds-base--inverse` provide? Can one or the other be used?